src/server/game/Battlefield/Zones/BattlefieldWG.h

```diff
@@ -408,6 +408,7 @@ class TC_GAME_API BattlefieldWG : public Battlefield
 
         uint32 m_tenacityStack;
         uint32 m_saveTimer;
+        uint8 m_buffedTeam;
```

Comment:
what's the value of this class member field when BattlefieldWG::UpdateTenacity() is called for the first time ?